Sources/LiveViewNative/Modifiers/Menus and Commands/MenuStyleModifier.swift

```diff
@@ -0,0 +1,70 @@
+//
+//  MenuStyleModifier.swift
+//  LiveViewNative
+//
+//  Created by Shadowfacts on 5/12/23.
+//
+
+import SwiftUI
+
+/// Applies a style to a ``Menu`` view.
+///
+/// The `button` style may be combined with the ``ButtonStyleModifier`` to alter the button's appearance.
```

Comment:
The symbol link will be transformed to `button_style` so it needs the word `modifier` after.
```suggestion
/// The `button` style may be combined with the ``ButtonStyleModifier`` modifier to alter the button's appearance.
```